exploration/maintaining-registry.md

```diff
@@ -230,28 +251,31 @@ In such cases, the `u:` namespace version is retained, but deprecated.
 > but it is not universally available and could represent a barrier to adoption
 > if normatively required.
 
-Any registry entry goes through a development process that includes these levels of maturity:
+All `Standard`, `Optional`, and Unicode namespace function or option specifications goes through 
+a development process that includes these levels of maturity:
```

Comment:
The list of levels below doesn't match the lists above, which don't include Rejected at all and only include Proposed and Accepted for Optional items.